lib/github_advisory_sync.rb

```diff
@@ -342,9 +342,10 @@ def create(package)
       filename_to_write = package.filename
 
       new_data = package.merge_data(
-        "cvss_v3"             => ("<FILL IN IF AVAILABLE>" unless cvss),
-        "patched_versions"    => ["<FILL IN SEE BELOW>"],
-        "unaffected_versions" => ["<OPTIONAL: FILL IN SEE BELOW>"]
+        "cvss_v3"             => (nil unless cvss),
+#HID        "cvss_v3"             => ("<FILL IN IF AVAILABLE>" unless cvss),
+#HID        "patched_versions"    => ["<FILL IN SEE BELOW>"],
+#HID        "unaffected_versions" => ["<OPTIONAL: FILL IN SEE BELOW>"]
```

Comment:
Did you test your changes with a GHSA advisory that had multiple `patched_versions` ranges? Such as [GHSA-6x85-j5j2-27jx](https://github.com/advisories/GHSA-6x85-j5j2-27jx). When a patched version range has an upper and lower bound (ex: `>= 2.1.0 and < 3.0.0`) then it needs to be represented as a comma separated YAML String (ex: `>= 2.1.0, < 3.0.0`) which `bundler-audit` can parse. I'm not sure if GitHub's GraphQL end-point returns `patched_versions` as an Array of Strings (ex: `[">= 2.1.0, < 3.0.0", ...]`) or possibly an Array of Arrays of Strings (ex: `[[">= 2.1.0", "< 3.0.0"], ...]`)?

If the new code changes work perfectly, you can simply delete the commented out code, as it's no longer needed.